### ISSUE TITLE
Documentation for $it

### ISSUE THREAD

**Issue by tjtnew** [nushell/book#98]
I may have missed something obvious but I can't seem to find any documentation of what `$it` is actually doing (although there are examples of it being used within pipelines).  Could anyone explain?  

Apologies if this should have been asked in the nushell repo but felt here was better.

Thankyou for your work on this.  It's great!

**Comment by thegedge** [comment]
Yep, we definitely lack information in the book on special variables. Definitely worth talking about 🙂
